src/Commands/Https/DeleteCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Https;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+class DeleteCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Delete or disable https for a site.
+     *
+     * @authorized
+     *
+     * @command https:delete
+     *
+     * @param string $site_env Site and environment in the form `site-name.env`.
+     *
+     * @usage terminus https:delete <site_name>.<env_name>
+     */
+    public function delete($site_env)
+    {
+        list(, $env) = $this->getSiteEnv($site_env, 'dev');
+        $workflow = $env->disableHttpsCertificate();
```

Comment:
Between the disabling of HTTPS and the initiation of the convergence workflow, there should be a notice emitted to the effect of HTTPS has been disabled and the environment's bindings will now be converged.